src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -157,32 +161,36 @@ private function validate(\Twig_Environment $twig, $template, $file)
         return array('template' => $template, 'file' => $file, 'valid' => true);
     }
 
-    private function display(InputInterface $input, OutputInterface $output, $files)
+    private function display(InputInterface $input, OutputInterface $stdout, $output, $files)
     {
         switch ($input->getOption('format')) {
             case 'txt':
-                return $this->displayTxt($output, $files);
+                return $this->displayTxt($stdout, $output, $files);
             case 'json':
-                return $this->displayJson($output, $files);
+                return $this->displayJson($stdout, $files);
             default:
                 throw new \InvalidArgumentException(sprintf('The format "%s" is not supported.', $input->getOption('format')));
         }
     }
 
-    private function displayTxt(OutputInterface $output, $filesInfo)
+    private function displayTxt(OutputInterface $stdout, $output, $filesInfo)
     {
         $errors = 0;
 
         foreach ($filesInfo as $info) {
-            if ($info['valid'] && $output->isVerbose()) {
-                $output->writeln('<info>OK</info>'.($info['file'] ? sprintf(' in %s', $info['file']) : ''));
+            if ($info['valid'] && $stdout->isVerbose()) {
```

Comment:
Same comment as https://github.com/symfony/symfony/pull/15991#discussion_r40708621
What do you think ?